src/scheduler/fifo.c

```diff
@@ -2355,6 +2351,46 @@ next_job(status *policy, server_info *sinfo, int flag)
 	return rjob;
 }
 
+/**
+ * @brief
+ *	Helper function used to copy the given source string to the destination string. It also
+ *	frees the destination and then allocates required memory before copying.
+ *
+ * @param[in] dest - Address of pointer to destination string
+ * @param[in] dest - pointer to source string
+ *
+ * @retval
+ * @return 0 - Failure
+ * @return  1 - Success
+ *
+ * @par Side Effects:
+ *	None
+ *
+ *
+ */
+static int
+copy_attr_value(char **dest, char *src)
+{
+	int len = 0;
+	int ret = 0;
+
+	if (*dest != NULL)
+		free(*dest);
+
+	if (src !=NULL) {
```

Comment:
Put a space after =

Reply:
Done

Reply:
Is this coming in a future commit?  I don't see it done.

Reply:
yes